backends/tofino/bf-p4c/common/extract_maupipe.cpp

```diff
@@ -322,7 +322,7 @@ class SetupActionProperties : public MauModifier {
             if (!path) BUG("Default action path %s cannot be found", default_action);
             auto actName = refMap->getDeclaration(path->path, true)->externalName();
 
-            if (actName == act->name) {
+            if (actName == act->name.name) {
```

Comment:
Why are all the redundant `.name` needed?  You should be able to compare an `IR::ID` to any other kind of string directly.

Reply:
Yet more gcc-11 breakage.  It is annoying that they never updated gcc to something supporting C++20 on ubuntu 22.04.